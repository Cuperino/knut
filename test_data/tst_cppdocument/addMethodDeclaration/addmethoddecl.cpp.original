class Student {

public:
  int count;
  
private:
  QString name;
  int rollNo;

};
