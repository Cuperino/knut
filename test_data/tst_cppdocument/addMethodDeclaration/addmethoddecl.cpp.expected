class Student {

public:
  int count;
  bool func(QString s, int a);
  
private:
  QString name;
  int rollNo;

protected:
  bool foo(QString s, int a);

};
